Formula/ocaml.rb

```diff
@@ -38,6 +38,15 @@ class Ocaml < Formula
     satisfy { HOMEBREW_PREFIX.to_s == "/usr/local" }
   end
```

Comment:
I think this needs to accept `/opt/homebrew` as prefix on ARM.

Reply:
I know… but it's not building anyway :(